test/ServerComparison.FunctionalTests/NtlmAuthenticationTest.cs

```diff
@@ -40,7 +40,8 @@ public class NtlmAuthenticationTests
                     EnvironmentName = "NtlmAuthentication", // Will pick the Start class named 'StartupNtlmAuthentication'
                     ServerConfigTemplateContent = Helpers.GetConfigContent(serverType, "NtlmAuthentication.config", nginxConfig: null),
                     SiteName = "NtlmAuthenticationTestSite", // This is configured in the NtlmAuthentication.config
-                    ApplicationType = applicationType
+                    ApplicationType = applicationType,
+                    TargetFramework = "net452",
```

Comment:
`SelfHostDeployer` (used for HttpSysServer) defaults to `"net451"`. We might be able to bump that default up once I'm done changing all the test projects.